spec/features/edit_user_story_spec.rb

```diff
@@ -11,12 +11,21 @@
 
   it "edits a user story" do
     visit "/projects/#{@project.id}/user_stories/#{@user_story.id}/edit"
-    fill_in 'user_story_definition', :with => 'As a user I want beef to eat'
+    fill_in 'user_story_definition', with: 'As a user I want beef to eat'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.